images/ceph/Makefile

```diff
@@ -18,9 +18,9 @@ include ../image.mk
 # Image Build Options
 
 ifeq ($(GOARCH),amd64)
-CEPH_VERSION ?= v18.2.4-20240723
+CEPH_VERSION ?= v19.2.0-20240927
```

Comment:
```suggestion
CEPH_VERSION ?= v19.2.1-20250202
```

`v19.2.1-20250202` is available for x86 and arm

Reply:
<img width="1522" alt="image" src="https://github.com/user-attachments/assets/d9c1a313-cd93-4dd3-95a9-25b24ea98701" />

This is the quay expanded view. The normal view is just the top line, but the two little blue penguin icons next to "6 days ago" mean that this is a manifest image and has 2 linux images as part of the manifest. Hovering over the icons shows the arch info. In the expanded view here, we can see that they are amd64 (x86) and arm64. 

I suspect Ceph stopped building to ceph/ceph-amd64 with their new build pipeline given that there are no 19.2.1 images there, but clearly the Ceph release has both images in its manifests.

Reply:
Updating the base image to squid is under investigation in #14792. Let's not worry about it with this PR, so we can unblock merging it.